### PR TITLE
Tab edit preview

### PR DESCRIPTION
**Pull request by alexrochas**
Just added a new tab to preview text format. As an issue, created to be like some technical debt, use javascript libs instead of create a new API endpoint.

What do you think? Please, make critics about what was done, and suggestions also. =P

ps: also added a docker-compose file to run redis and mongo.

### CODE REVIEW

File: src/main/groovy/adamatti/web/RestView.groovy [hunk 1]
```diff
@@ -1,7 +1,9 @@
 package adamatti.web
-
+import adamatti.bizo.TiddlerRenderBO
 import adamatti.commons.TemplateHelper
 import adamatti.model.dao.TiddlerDAO
+import adamatti.model.entity.Tiddler
+import com.newrelic.agent.deps.com.google.common.collect.ImmutableMap
```
Comment (adamatti) [inline src/main/groovy/adamatti/web/RestView.groovy]: Hmm... ImmutableMap from newrelic?

**Review by adamatti — COMMENTED** [review]
All things are good, except by the ImmutableMap from newrelic. NewRelic should be used for instrumentation only and not in the main code

File: src/main/groovy/adamatti/web/RestView.groovy (hunk 1 above)
Comment (alexrochas) [reply src/main/groovy/adamatti/web/RestView.groovy]: You are right, my bad! 

Will change this, I'm habituated to use ImmutableMap from java8 std lib.

**Comment by alexrochas** [comment]
Dude, if I google it before...it's so simple create a map key/value.

There is! Waiting your feedback. =)

**Pull request merged**